Handle divorces

elysch opened the pull request.

Neriderc commented:
Thanks for this! Apologies I have not got to this until now. I will try to find time in the next day or two.

I notice some little things from an initial look. No option not to show divorces that I can see, divorce prefix doesn't have an option in the admin panel. 

I can make these changes as I review the pull request, and try to make sure things are all aligned with other options.

Thanks again for this pull request :slightly_smiling_face:

elysch commented:
I added the option for divorce_prefix to the admin area.

You're right, there is no option to prevent divorces to be shown... but also with marriages there is no such option. Maybe that's ok.

I'll try to add it for divorces.

P.S. I added another pull request with already merged this one and the one about a label about pedigree in the arrow from family to individual. That's to make thinks easier for you.

Neriderc commented:
> You're right, there is no option to prevent divorces to be shown... but also with marriages there is no such option. Maybe that's ok.

All items are hidable. If you uncheck all the options about marriage (show marriage date and place) then it still shows the family circle but no marriage details are show. It is possible to hide all info and only have a diagram of circles and squares.

> P.S. I added another pull request with already merged this one and the one about a label about pedigree in the arrow from family to individual. That's to make thinks easier for you.

That includes all your changes? So I should close this PR and also #569 then only look at #570 for inclusion?

elysch commented:
Yes
